[2024-04-04T00:05:35.159+0000] {processor.py:161} INFO - Started process (PID=99198) to work on /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
[2024-04-04T00:05:35.161+0000] {processor.py:830} INFO - Processing file /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py for tasks to queue
[2024-04-04T00:05:35.163+0000] {logging_mixin.py:188} INFO - [2024-04-04T00:05:35.162+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
[2024-04-04T00:05:35.486+0000] {logging_mixin.py:188} INFO - [2024-04-04T00:05:35.478+0000] {dagbag.py:350} ERROR - Failed to import: /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
Traceback (most recent call last):
  File "/home/datatalks_jan/anaconda3/lib/python3.11/site-packages/airflow/utils/deprecation_tools.py", line 56, in getattr_with_deprecation
    return getattr(importlib.import_module(new_module), new_class_name)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/datatalks_jan/anaconda3/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1126, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1126, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1140, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'airflow.providers.docker'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/datatalks_jan/anaconda3/lib/python3.11/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py", line 4, in <module>
    from airflow.operators.docker_operator import DockerOperator
  File "/home/datatalks_jan/anaconda3/lib/python3.11/site-packages/airflow/utils/deprecation_tools.py", line 63, in getattr_with_deprecation
    raise ImportError(error_message) from e
ImportError: Could not import `airflow.providers.docker.operators.docker.DockerOperator` while trying to import `airflow.operators.docker_operator.DockerOperator`.
[2024-04-04T00:05:35.487+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
[2024-04-04T00:05:35.524+0000] {processor.py:183} INFO - Processing /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py took 0.373 seconds
[2024-04-04T00:06:41.613+0000] {processor.py:161} INFO - Started process (PID=99577) to work on /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
[2024-04-04T00:06:41.615+0000] {processor.py:830} INFO - Processing file /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py for tasks to queue
[2024-04-04T00:06:41.617+0000] {logging_mixin.py:188} INFO - [2024-04-04T00:06:41.616+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
[2024-04-04T00:06:41.803+0000] {logging_mixin.py:188} INFO - [2024-04-04T00:06:41.799+0000] {dagbag.py:350} ERROR - Failed to import: /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
Traceback (most recent call last):
  File "/home/datatalks_jan/anaconda3/lib/python3.11/site-packages/airflow/utils/deprecation_tools.py", line 56, in getattr_with_deprecation
    return getattr(importlib.import_module(new_module), new_class_name)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/datatalks_jan/anaconda3/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1126, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1126, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1140, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'airflow.providers.docker'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/datatalks_jan/anaconda3/lib/python3.11/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py", line 4, in <module>
    from airflow.operators.docker_operator import DockerOperator
  File "/home/datatalks_jan/anaconda3/lib/python3.11/site-packages/airflow/utils/deprecation_tools.py", line 63, in getattr_with_deprecation
    raise ImportError(error_message) from e
ImportError: Could not import `airflow.providers.docker.operators.docker.DockerOperator` while trying to import `airflow.operators.docker_operator.DockerOperator`.
[2024-04-04T00:06:41.804+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
[2024-04-04T00:06:42.028+0000] {processor.py:183} INFO - Processing /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py took 0.420 seconds
[2024-04-04T00:07:14.377+0000] {processor.py:161} INFO - Started process (PID=99835) to work on /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
[2024-04-04T00:07:14.379+0000] {processor.py:830} INFO - Processing file /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py for tasks to queue
[2024-04-04T00:07:14.380+0000] {logging_mixin.py:188} INFO - [2024-04-04T00:07:14.379+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
[2024-04-04T00:07:14.885+0000] {logging_mixin.py:188} INFO - [2024-04-04T00:07:14.884+0000] {dagbag.py:350} ERROR - Failed to import: /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
Traceback (most recent call last):
  File "/home/datatalks_jan/anaconda3/lib/python3.11/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py", line 32, in <module>
    'retry_delay': timedelta(minutes=5),
                   ^^^^^^^^^
NameError: name 'timedelta' is not defined
[2024-04-04T00:07:14.886+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
[2024-04-04T00:07:14.911+0000] {processor.py:183} INFO - Processing /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py took 0.539 seconds
[2024-04-04T00:07:48.264+0000] {processor.py:161} INFO - Started process (PID=99967) to work on /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
[2024-04-04T00:07:48.266+0000] {processor.py:830} INFO - Processing file /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py for tasks to queue
[2024-04-04T00:07:48.268+0000] {logging_mixin.py:188} INFO - [2024-04-04T00:07:48.267+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
[2024-04-04T00:07:48.761+0000] {logging_mixin.py:188} INFO - [2024-04-04T00:07:48.759+0000] {dagbag.py:350} ERROR - Failed to import: /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
Traceback (most recent call last):
  File "/home/datatalks_jan/anaconda3/lib/python3.11/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py", line 34, in <module>
    'retry_delay': timedelta(minutes=5),
                   ^^^^^^^^^
NameError: name 'timedelta' is not defined
[2024-04-04T00:07:48.762+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
[2024-04-04T00:07:48.781+0000] {processor.py:183} INFO - Processing /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py took 0.524 seconds
[2024-04-04T00:08:22.859+0000] {processor.py:161} INFO - Started process (PID=100251) to work on /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
[2024-04-04T00:08:22.862+0000] {processor.py:830} INFO - Processing file /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py for tasks to queue
[2024-04-04T00:08:22.863+0000] {logging_mixin.py:188} INFO - [2024-04-04T00:08:22.863+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
[2024-04-04T00:08:23.367+0000] {processor.py:840} INFO - DAG(s) 'conrad_coding' retrieved from /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
[2024-04-04T00:08:23.476+0000] {logging_mixin.py:188} INFO - [2024-04-04T00:08:23.475+0000] {override.py:1769} INFO - Created Permission View: can read on DAG:conrad_coding
[2024-04-04T00:08:23.494+0000] {logging_mixin.py:188} INFO - [2024-04-04T00:08:23.493+0000] {override.py:1769} INFO - Created Permission View: can edit on DAG:conrad_coding
[2024-04-04T00:08:23.505+0000] {logging_mixin.py:188} INFO - [2024-04-04T00:08:23.504+0000] {override.py:1769} INFO - Created Permission View: can delete on DAG:conrad_coding
[2024-04-04T00:08:23.506+0000] {logging_mixin.py:188} INFO - [2024-04-04T00:08:23.505+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-04-04T00:08:23.517+0000] {logging_mixin.py:188} INFO - [2024-04-04T00:08:23.516+0000] {dag.py:3069} INFO - Creating ORM DAG for conrad_coding
[2024-04-04T00:08:23.527+0000] {logging_mixin.py:188} INFO - [2024-04-04T00:08:23.526+0000] {dag.py:3834} INFO - Setting next_dagrun for conrad_coding to 2024-04-07 00:00:00+00:00, run_after=2024-04-14 00:00:00+00:00
[2024-04-04T00:08:23.552+0000] {processor.py:183} INFO - Processing /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py took 0.699 seconds
[2024-04-04T00:08:56.323+0000] {processor.py:161} INFO - Started process (PID=100372) to work on /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
[2024-04-04T00:08:56.324+0000] {processor.py:830} INFO - Processing file /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py for tasks to queue
[2024-04-04T00:08:56.325+0000] {logging_mixin.py:188} INFO - [2024-04-04T00:08:56.325+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
[2024-04-04T00:08:56.740+0000] {processor.py:840} INFO - DAG(s) 'conrad_coding' retrieved from /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
[2024-04-04T00:08:56.754+0000] {logging_mixin.py:188} INFO - [2024-04-04T00:08:56.753+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-04-04T00:08:56.772+0000] {logging_mixin.py:188} INFO - [2024-04-04T00:08:56.772+0000] {dag.py:3834} INFO - Setting next_dagrun for conrad_coding to 2024-04-07 00:00:00+00:00, run_after=2024-04-14 00:00:00+00:00
[2024-04-04T00:08:56.795+0000] {processor.py:183} INFO - Processing /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py took 0.477 seconds
[2024-04-04T00:09:32.158+0000] {processor.py:161} INFO - Started process (PID=100621) to work on /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
[2024-04-04T00:09:32.161+0000] {processor.py:830} INFO - Processing file /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py for tasks to queue
[2024-04-04T00:09:32.164+0000] {logging_mixin.py:188} INFO - [2024-04-04T00:09:32.162+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
[2024-04-04T00:09:32.479+0000] {processor.py:840} INFO - DAG(s) 'conrad_coding' retrieved from /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
[2024-04-04T00:09:32.505+0000] {logging_mixin.py:188} INFO - [2024-04-04T00:09:32.504+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-04-04T00:09:32.534+0000] {logging_mixin.py:188} INFO - [2024-04-04T00:09:32.533+0000] {dag.py:3834} INFO - Setting next_dagrun for conrad_coding to 2024-04-07 00:00:00+00:00, run_after=2024-04-14 00:00:00+00:00
[2024-04-04T00:09:32.568+0000] {processor.py:183} INFO - Processing /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py took 0.415 seconds
[2024-04-04T00:09:59.745+0000] {processor.py:161} INFO - Started process (PID=100862) to work on /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
[2024-04-04T00:09:59.746+0000] {processor.py:830} INFO - Processing file /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py for tasks to queue
[2024-04-04T00:09:59.747+0000] {logging_mixin.py:188} INFO - [2024-04-04T00:09:59.747+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
[2024-04-04T00:09:59.982+0000] {processor.py:840} INFO - DAG(s) 'conrad_coding' retrieved from /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
[2024-04-04T00:10:00.001+0000] {logging_mixin.py:188} INFO - [2024-04-04T00:10:00.001+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-04-04T00:10:00.024+0000] {logging_mixin.py:188} INFO - [2024-04-04T00:10:00.024+0000] {dag.py:3834} INFO - Setting next_dagrun for conrad_coding to 2024-04-07 00:00:00+00:00, run_after=2024-04-14 00:00:00+00:00
[2024-04-04T00:10:00.052+0000] {processor.py:183} INFO - Processing /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py took 0.312 seconds
[2024-04-04T00:10:57.271+0000] {processor.py:161} INFO - Started process (PID=101389) to work on /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
[2024-04-04T00:10:57.273+0000] {processor.py:830} INFO - Processing file /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py for tasks to queue
[2024-04-04T00:10:57.275+0000] {logging_mixin.py:188} INFO - [2024-04-04T00:10:57.274+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
[2024-04-04T00:10:57.491+0000] {processor.py:840} INFO - DAG(s) 'conrad_coding' retrieved from /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
[2024-04-04T00:10:57.516+0000] {logging_mixin.py:188} INFO - [2024-04-04T00:10:57.516+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-04-04T00:10:57.539+0000] {logging_mixin.py:188} INFO - [2024-04-04T00:10:57.539+0000] {dag.py:3834} INFO - Setting next_dagrun for conrad_coding to 2024-04-07 00:00:00+00:00, run_after=2024-04-14 00:00:00+00:00
[2024-04-04T00:10:57.563+0000] {processor.py:183} INFO - Processing /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py took 0.297 seconds
[2024-04-04T00:11:44.828+0000] {processor.py:161} INFO - Started process (PID=101706) to work on /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
[2024-04-04T00:11:44.831+0000] {processor.py:830} INFO - Processing file /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py for tasks to queue
[2024-04-04T00:11:44.833+0000] {logging_mixin.py:188} INFO - [2024-04-04T00:11:44.832+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
[2024-04-04T00:11:45.072+0000] {processor.py:840} INFO - DAG(s) 'conrad_coding' retrieved from /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
[2024-04-04T00:11:45.097+0000] {logging_mixin.py:188} INFO - [2024-04-04T00:11:45.096+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-04-04T00:11:45.127+0000] {logging_mixin.py:188} INFO - [2024-04-04T00:11:45.126+0000] {dag.py:3834} INFO - Setting next_dagrun for conrad_coding to 2024-04-07 00:00:00+00:00, run_after=2024-04-14 00:00:00+00:00
[2024-04-04T00:11:45.155+0000] {processor.py:183} INFO - Processing /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py took 0.333 seconds
[2024-04-04T00:12:09.152+0000] {processor.py:161} INFO - Started process (PID=101894) to work on /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
[2024-04-04T00:12:09.154+0000] {processor.py:830} INFO - Processing file /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py for tasks to queue
[2024-04-04T00:12:09.155+0000] {logging_mixin.py:188} INFO - [2024-04-04T00:12:09.155+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
[2024-04-04T00:12:09.162+0000] {logging_mixin.py:188} INFO - [2024-04-04T00:12:09.160+0000] {dagbag.py:350} ERROR - Failed to import: /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
Traceback (most recent call last):
  File "/home/datatalks_jan/anaconda3/lib/python3.11/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 936, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1074, in get_code
  File "<frozen importlib._bootstrap_external>", line 1004, in source_to_code
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py", line 19
    """
    ^
SyntaxError: unterminated triple-quoted string literal (detected at line 50)
[2024-04-04T00:12:09.163+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
[2024-04-04T00:12:09.192+0000] {processor.py:183} INFO - Processing /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py took 0.045 seconds
[2024-04-04T00:12:57.749+0000] {processor.py:161} INFO - Started process (PID=102183) to work on /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
[2024-04-04T00:12:57.752+0000] {processor.py:830} INFO - Processing file /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py for tasks to queue
[2024-04-04T00:12:57.753+0000] {logging_mixin.py:188} INFO - [2024-04-04T00:12:57.753+0000] {dagbag.py:540} INFO - Filling up the DagBag from /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
[2024-04-04T00:12:57.973+0000] {processor.py:840} INFO - DAG(s) 'conrad_coding' retrieved from /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py
[2024-04-04T00:12:57.992+0000] {logging_mixin.py:188} INFO - [2024-04-04T00:12:57.992+0000] {dag.py:3047} INFO - Sync 1 DAGs
[2024-04-04T00:12:58.017+0000] {logging_mixin.py:188} INFO - [2024-04-04T00:12:58.017+0000] {dag.py:3834} INFO - Setting next_dagrun for conrad_coding to 2024-04-07 00:00:00+00:00, run_after=2024-04-14 00:00:00+00:00
[2024-04-04T00:12:58.048+0000] {processor.py:183} INFO - Processing /home/datatalks_jan/Data_Eden/3_airflow_demo/airflow/dags/dag_spark_multi_source_ETL_new.py took 0.308 seconds
